tensornetwork/linalg/linalg_test.py

```diff
@@ -5,42 +5,63 @@
 import jax.config as config
 import torch
 import tensorflow as tf
+import tensornetwork as tn
```

Comment:
importing tensornetwork directly should only happen in tests that fall in `tensornetwork/tests`.